tox.ini

```diff
@@ -46,10 +46,9 @@ commands =
 [testenv:docs]
 skip_install = True
 deps =
-    -r{toxinidir}/docs/requirements.txt
-    ./core
+    ./core[docs]
 commands =
-    python -m sphinx -W docs build/sphinx
+    python -m sphinx -W docs docs/_build
```

Comment:
Now that the `toga` meta-package has its own subdirectory, there's no longer any need to clutter the repository root with a `build` directory. Of course, it's a gitignored directory, so you'll have to delete it manually in your own local copy.